filetrust/cdr-plugin-folder-to-folder
# Update .env.sample to reflect proper value of root_folder

## Summary

The default value of root_folder contains scenario name and that is used for unit testing. We can override the default value using .env.sample which is copied as a .env into the API container and this should enable config endpoint to return proper value of the root_folder.

Now proper value can be used by UI to dynamically build the proper paths for the scenarios

Also updated the Cloud SDK endpoints to use LB address instead of list of endpoints

## Testing

1. Start the docker-compose
2. Check the configuration endpoint to see the value of root_folder is /app/test_data

## Bug
https://github.com/filetrust/cdr-plugin-folder-to-folder-bugs/issues/67
